wezterm-gui/src/termwindow/render/fancy_tab_bar.rs

```diff
@@ -288,6 +288,7 @@ impl crate::TermWindow {
                     &font,
                     &metrics,
                     &self.config,
+                    tab_bar_height,
```

Comment:
this isn't the right approach. The font context and render metrics contain sufficient information about the size and dpi that the existing poly drawing stuff can use to scale appropriately.

Probably the right way to resolve this is to switch to using Points instead of Pixels in the gnome logic, similar to the way that the Windows flavor of these decorations works.